### PR TITLE
Moved: ByteDance.DouyinIDE version 3.0.9

### PR DESCRIPTION
**Pull request by SpecterShell**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

### CODE REVIEW

**Review by ItzLevvie — DISMISSED** [review]

File: manifests/b/ByteDance/DouyinIDE/3.0.9/ByteDance.DouyinIDE.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://sf3-ttcdn-tos.pstatp.com/obj/developer/ide/5954052/win32/%E5%AD%97%E8%8A%82%E8%B7%B3%E5%8A%A8%E5%BC%80%E5%8F%91%E8%80%85%E5%B7%A5%E5%85%B7-3.0.9.exe
+  InstallerSha256: ED46CB7047908BE269220325BF4D33C104EDC4735362EAEB801ACC6F5A222CA9
+  InstallerSwitches:
+    Custom: /currentuser
+  ProductCode: 0d6cc698-a1aa-5593-a60b-0310adcebfe2
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://sf3-ttcdn-tos.pstatp.com/obj/developer/ide/5954052/win32/%E5%AD%97%E8%8A%82%E8%B7%B3%E5%8A%A8%E5%BC%80%E5%8F%91%E8%80%85%E5%B7%A5%E5%85%B7-3.0.9.exe
```
Comment (stephengillie) [inline manifests/b/ByteDance/DouyinIDE/3.0.9/ByteDance.DouyinIDE.installer.yaml:28]:
```suggestion
  InstallerUrl: https://sf1-cdn-tos.douyinstatic.com/obj/microapp/frontend/ide/5954052/win32/%E5%AD%97%E8%8A%82%E8%B7%B3%E5%8A%A8%E5%BC%80%E5%8F%91%E8%80%85%E5%B7%A5%E5%85%B7-3.0.9.exe
  InstallerSha256: ED46CB7047908BE269220325BF4D33C104EDC4735362EAEB801ACC6F5A222CA9
  InstallerSwitches:
    Custom: /currentuser
  ProductCode: 0d6cc698-a1aa-5593-a60b-0310adcebfe2
- Architecture: x64
  Scope: machine
  InstallerUrl: https://sf1-cdn-tos.douyinstatic.com/obj/microapp/frontend/ide/5954052/win32/%E5%AD%97%E8%8A%82%E8%B7%B3%E5%8A%A8%E5%BC%80%E5%8F%91%E8%80%85%E5%B7%A5%E5%85%B7-3.0.9.exe
```

Comment (stephengillie) [reply manifests/b/ByteDance/DouyinIDE/3.0.9/ByteDance.DouyinIDE.installer.yaml:28]: Hi @SpecterShell,

The installer has to come from the developer's website.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**